api/envoy/config/bootstrap/v3/bootstrap.proto

```diff
@@ -208,6 +209,11 @@ message ClusterManager {
 
     // Specifies the path to the outlier event log.
     string event_log_path = 1;
+
+    // [#not-implemented-hide:]
+    // The gRPC service for the outlier detection event service.
+    // If empty, outlier detection events won't be sent to a remote endpoint.
```

Comment:
Might want a section in the docs on how the event streaming service works, but it can also wait until implementation.

Reply:
yes, I also think it's better to write when some implementation will be in place.